src/helm/proxy/clients/critique_client.py

```diff
@@ -308,15 +401,26 @@ def _critique_question_to_scale_field(self, question: CritiqueQuestionTemplate,
                 "title": question.name,
                 "description": self._interpolate_fields(question.text, fields),
                 "choices": [{"label": option, "value": option} for option in question.options],
+                "min_choices": 1,
+                "max_choices": len(question.options) if question.question_type == "checkbox" else 1,
             }
         else:
             raise ValueError(f"Unsupported question type {question.question_type}")
 
     def _get_or_create_scale_task(
-        self, project_name: str, template: CritiqueTaskTemplate, fields: Dict[str, str]
+        self, project_name: str, batch_name: str, template: CritiqueTaskTemplate, fields: Dict[str, str]
     ) -> str:
         """Get or create a task on Scale and return the Scale task ID."""
 
+        # Used both for the cache key and the task unique_id
+        cache_key = {
+            "testing": True,
+            "project": project_name,
```

Comment:
Don't specify `project`.

https://docs.scale.com/reference/text-collection

"Note that if a batch is specified, you need not specify the project, as the task will automatically be associated with the batch's project."